### ISSUE TITLE
add `pull_parsnip_fit()`

### ISSUE THREAD

**Issue by topepo** [tidymodels/parsnip#469]
To pluck `fit` from a `parsnip` object.

**Comment by EmilHvitfeldt** [comment]
Will this have a related function in workflows?

**Comment by topepo** [comment]
Already [here](https://workflows.tidymodels.org/reference/workflow-extractors.html) which makes me think that we need to advertise these more.

**Comment by EmilHvitfeldt** [comment]
So the expected code to extract the modelfit from a fitted workflow is

``` r
library(tidymodels)

lm_spec <- linear_reg() %>%
  set_engine("lm")

rec_spec <- recipe(mpg ~ ., data = mtcars)

wf_spec <- workflow() %>%
  add_recipe(rec_spec) %>%
  add_model(lm_spec)

wf_fit <- fit(wf_spec, data = mtcars)

wf_fit %>%
  pull_workflow_fit() %>%
  pull_parsnip_fit()
#> 
#> Call:
#> stats::lm(formula = ..y ~ ., data = data)
#> 
#> Coefficients:
#> (Intercept)          cyl         disp           hp         drat           wt  
#>    12.30337     -0.11144      0.01334     -0.02148      0.78711     -3.71530  
#>        qsec           vs           am         gear         carb  
#>     0.82104      0.31776      2.52023      0.65541     -0.19942
```

<sup>Created on 2021-04-12 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

**Comment by topepo** [comment]
Yes, but that makes me think that I should rename it to something like `pull_model_object()` or similar.

**Comment by EmilHvitfeldt** [comment]
Yes, the naming convention is not the best here. From seeing the name alone I would expect `pull_parsnip_fit()` to do what `pull_workflow_fit()` is doing

**Comment by juliasilge** [comment]
Closed in #518
